site/_js/analytics.js

```diff
@@ -47,7 +47,14 @@ function sendToGoogleAnalytics({name, delta, id, attribution, navigationType}) {
 
     // See: https://web.dev/debug-web-vitals-in-the-field/
     [dimensions.WEB_VITALS_DEBUG]: webVitalInfo,
-    [dimensions.NAVIGATION_TYPE]: navigationType,
+    // Override for 'navigational-prefetch' for the prefetch origin trial
+    // experiment (https://github.com/GoogleChrome/web.dev/pull/9532)
+    [dimensions.NAVIGATION_TYPE]:
+      navigationType === 'navigate' &&
+      performance.getEntriesByType('navigation')[0].deliveryType ===
```

Comment:
just checking whether or not you want to feature-detect the performance timeline API or not (admittedly it's been some time, but web.dev did in one place)

Reply:
Yeah added that.